geopy/distance.py

```diff
@@ -79,6 +102,17 @@
 from geopy.point import Point
 from geopy.compat import string_compare
 
+HAVE_PROJ = HAVE_GEODESIC = False
+try:
+    from pyproj import Geod
+    HAVE_PROJ = HAVE_GEODESIC = True
```

Comment:
I personally don't like having booleans there. Take this line for example: `geographiclib` presence isn't even checked, though the boolean claims that it's there.

I'm good with booleans if these imports are tested individually, but in this case we would have a redundant import of `geographiclib` when `pyproj` is installed.

Maybe using string is a better choice there? Like this:
```
try:
    from pyproj import Geod
    GEODESIC_METHOD = 'pyproj'
except ImportError:
    try:
        from geographiclib.geodesic import Geodesic
        GEODESIC_METHOD = 'geographiclib'
    except ImportError:
        GEODESIC_METHOD = None  # not available
```